.all-contributorsrc

```diff
@@ -0,0 +1,322 @@
+{
+  "projectName": "wing",
+  "projectOwner": "winglang",
```

Comment:
Do we know who the project owner is or do we want to keep as winglang?

Reply:
I think we can keep winglang for now 👍